docs/core-concepts/core-sdk/cli/dotnet-pack.md

```diff
@@ -0,0 +1,80 @@
+dotnet-pack
+===========
+
+## NAME
+
+`dotnet-pack` - packs the code into a NuGet package
+
+## SYNOPSIS
+
+dotnet-pack [--output]  
+    [--no-build] [--build-base-path]  
+    [--configuration]  [--version-suffix]
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-pack` will build the project and create NuGet packages. The result of this operation are two packages 
```

Comment:
The result of this operation are two packages with the extension of  `nupkg`. -> The result of this operation **is** two packages with the `nupkg` extension.